task5/src/main/java/com/clouway/adapter/http/LogoutController.java

```diff
@@ -0,0 +1,35 @@
+package com.clouway.adapter.http;
+
+import com.clouway.core.OnlineUsers;
+import com.clouway.core.SessionRepository;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 02.06.16.
+ */
+public class LogoutController extends HttpServlet {
+  private SessionRepository sessionRepository;
+
+  public LogoutController(SessionRepository sessionRepository) {
+    this.sessionRepository = sessionRepository;
+  }
+
+  @Override
+  protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
Here at logout you delete the cookie but leave the session in db. What would you do to clean the session that left in DB after logOut?